exercises/two-fer/TwoFerTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+namespace TwoFer
+{
+    public class TwoFerTest
```

Comment:
The test class looks good, but it does not match the [canonical-data](https://github.com/exercism/problem-specifications/blob/master/exercises/two-fer/canonical-data.json). As far as I can tell, this exercise is intended to introduce the concept of optional/default values. I think it is perfect for that, so I would like to ask you to use the same test cases as the canonical data, where the "no name given" test case should call `TwoFer.GetResponse()` (without an argument). The implementation could then assign the string parameter a default value of `null`.